### DISCUSSION TITLE
SEB For Remote Tests Hosted on a Website

### DISCUSSION THREAD

**Discussion by trivikramns** [SafeExamBrowser/seb-mac#253] [Q&A]
Hi Team, Is there a way we can encrypt the .seb file or add a field with the hash of content in seb. We are using our custom seb generator to create the configuration for the seb, So anything by which we can add the field and Safe Exam Brower autodetects if file is hampered or not.
Any documentation for this would help.

**Comment by danschlet** [comment]
Just study safeexambrowser.org, particularly https://safeexambrowser.org/developer/overview.html.
